CHANGELOG.rst

```diff
@@ -42,6 +42,11 @@ Added
 
   Contributed by @Kami.
 
+* Fix a bug in the pack config loader so that objects covered by an additionalProperties schema
```

Comment:
Please also add the PR reference here - aka #5225

Reply:
done